services/webhook/tests/test_persona_evaluate.py

```
@@ -159,19 +159,19 @@ def test_evaluate_mixed_advisory_and_blocking_failure():
     assert scope.passed is False
     assert link.passed is False
     title, summary = persona._summary(list(evaluation.results))
-    assert "1/5 blocking" in title  # only scope-fence counts, not issue-link
+    assert title == "Hunt Plan hold - 1/5 plan checks fail"
 
 
 def test_summary_advisory_check_renders_warning_icon():
-    """Advisory checks that fail should render ⚠️ not ❌ in the summary."""
+    """Advisory checks that fail should render warn not fail in the summary."""
     results = [
         CheckResult("why", True, "ok"),
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)